service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -444,27 +437,30 @@ public void listingValidRegionsOnAPolicy() throws Exception {
   }
 
   @Pact(consumer = "wsm", provider = "tps")
-  public RequestResponsePact explainingAWorkspacePolicy(PactDslWithProvider builder) {
+  public V4Pact explainingAWorkspacePolicy(PactBuilder builder) {
     var explainResponse =
         new PactDslJsonBody()
             .numberType("depth")
-            .valueFromProviderState(
-                "objectId", existingPolicyProviderStateValue, existingPolicyId.toString())
+            .stringValue("objectId", existingPolicyId.toString())
             .object(
                 "policyInput",
-                new PactDslJsonBody().stringType("namespace").stringType("name").closeObject());
+                newJsonBody(
```

Comment:
I switched over to use the new DSL-sugary version of `PactdslJsonBody()` because the new Kotlin implementation has strict expectations about `null` which aren't satisfied by the old style.  I'm not sure why I didn't have to rewrite _all_ of the `PactDslJsonBody()` stuff in the file, but am happy that I got to avoid doing so.